Parser/Device/AbstractDeviceParser.php

```diff
@@ -592,7 +587,6 @@ abstract class AbstractDeviceParser extends AbstractParser
         'E7'  => 'Ergo',
         'EC'  => 'Ericsson',
         '05'  => 'Erisson',
-        'ER'  => 'Ericy',
```

Comment:
It might be good to keep such records, even if they are not used anymore. Otherwise the short codes might get used for something else in the future, causing an incorrect mapping for old data.
For those records that were merged with others we could change the name to be the same one and add a comment explaining it.